src/games/gameLogic.ts

```diff
@@ -1,64 +1,53 @@
-const board1 = [
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0']
-]
+import { ValidatorConstraint, ValidatorConstraintInterface } from 'class-validator'
+import { Board, emptyBoard, Row} from './entities';
 
-const board2 = [
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0'],
-  ['0', '0', '0', '0', '0', '0', '0', '0', '0', '0']
-]
+@ValidatorConstraint()
```

Comment:
do we need to define this if we already know how our board looks like? As in we know how big those arrays are. Not sure I understand this class completely

Reply:
The first reason I putted it here is that I saw it on the starter example haha!
I think it s just to make sure that we have the right thing. Maybe later on we create a new board and forgot we need 10 by 10 :smiley: , also we need strings as values

Reply:
ah alright. yeah then let's keep it here but keep in mind to check later on if we need it or not

Reply:
Sure :+1: